packages/sanity/src/core/perspective/navbar/GlobalPerspectiveMenuItem.tsx

```diff
@@ -135,7 +144,7 @@ export const GlobalPerspectiveMenuItem = forwardRef<
 
   return (
     <GlobalPerspectiveMenuItemIndicator
-      $isDraft={isDraftPerspective(release)}
+      $isDefaultPerspective={release === defaultPerspective}
```

Comment:
We repeat this at least twice so it might make sense to set this in a const (And outside of the render)